src/index.html

```diff
@@ -2,13 +2,72 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport" content="width=device-width, user-scalable=no, initial-scale=1,maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a class="img__link" href="#home">
+        <img
+          src="images/logo.png"
+          class="moyo__logo"
+          alt="moyoLogo"
```

Comment:
this text will available for users

```suggestion
          alt="Moyo logo"
```